pages/ksphere/konvoy/1.4/quick-start/index.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: layout.pug
+navigationTitle: Quick Start
+title: Quick Start
+menuWeight: 3
+excerpt: Get started by installing a cluster with default configuration settings
+enterprise: false
+---
+
+<!-- markdownlint-disable MD025 -->
+
+Konvoy is a tool for provisioning Kubernetes clusters with a suite of pre-selected [Cloud Native Computing Foundation (CNCF)][cncf] and community-contributed tools.
+By combining a native Kubernetes cluster as its foundation with a default set of cluster extensions,
+Konvoy provides a complete out-of-the-box solution for organizations that want to deploy production-ready
+Kubernetes.
+
+This Quick Start guide provides simplified instructions to get your Konvoy cluster up and running with minimal configuration requirements on an Amazon Web Services (AWS) public cloud instance.
+
+# Prerequisites
+
+All Konvoy runtime dependencies are bundled in a Docker container and packaged with a wrapper that executes the container and manages these dependencies.
+
+Before starting the Konvoy installation, you should verify the following:
+
+-   You have a Linux or MacOS computer with a supported version of the operating system.
+-   You have the [aws][install_aws] command-line utility if you are installing on an AWS cloud instance.
+-   You have [Docker Desktop][install_docker] version 18.09.2 or later.
+-   You have [kubectl][install_kubectl] v1.16.8 or later for interacting with the running cluster.
+-   You have a valid AWS account with [credentials configured][aws_credentials].
+    You must be authorized to create the following resources in the AWS account:
+    - EC2 Instances
+    - VPC
+    - Subnets
+    - Elastic Load Balancer (ELB)
+    - Internet Gateway
+    - NAT Gateway
+    - Elastic Block Storage (EBS) Volumes
+    - Security Groups
+    - Route Tables
+    - IAM Roles
+
+# Installing Konvoy
+
+1.  Install required packages. In most cases, you can install the required software using your preferred package manager. For example, on a macOS computer, you can use [Homebrew][brew] to install `kubectl` and the `aws` command-line utility by running the following command:
+
+    ```bash
+    brew install kubernetes-cli awscli
+    ```
+
+1.  Check the Kubernetes client version. Many important Kubernetes functions **do not work** if your client is outdated. You can verify that the version of `kubectl` you have installed is supported by running the following command:
+
+    ```bash
+    kubectl version --short=true`
+    ```
+
+1.  Download and extract the Konvoy package. You will need to download and extract the Konvoy package tarball. There are a couple of ways to get the tarball:
+
+    - You can request a free trial from D2iQ [here](https://d2iq.com/solutions/ksphere/konvoy#request-free-trial). We will send you a link for the tarball.
+    - If you are a D2iQ customer, you can ask your account representative for a trial license.
```

Comment:
If they are an existing Konvoy customer they can download it from the support portal (support.d2iq.com)

Reply:
Yeah, this should match the other pages where we talk about this. Or possibly, we should get rid of these references and just point to the master topic. I'd say don't hold this PR for that though, let's open a JIRA>